docker/arm-server/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This shell script exists for building ARM64 docker image.
+# Docker image includes HO(Host Orchestrator) inside,
+# so it could execute CF instance with API in HO.
+
+# Build debian packages(e.g. cuttlefish-base) with docker.
+pushd ..
+./debs-builder-docker/build-debs-with-docker.sh
```

Comment:
I thought we can still build debian package inside docker. maybe we can provide options: one for local source tree, the other for remote source tree.

Reply:
Doesn't `build-debs-with-docker.sh` build debian packages like cuttlefish-base, cuttlefish-user, or cuttlefish-orchestration inside Docker?

- Docker image X(debs-builder-docker/build-debs-with-docker.sh) builds CF debian packages.
- Build docker image Y(arm-server/build.sh) with installing CF debian packages on top of linux.
- Run docker image Y to run CF instance inside.

I thought we already build debian packages inside docker, no?

Reply:
But there is separate dockerfile for X. can we use https://docs.docker.com/build/building/multi-stage/ to combine them?

Reply:
Ack